src/python/test/test_dxclient.py

```diff
@@ -2728,6 +2728,33 @@ def main():
         shell.close()
         self.assertEqual(3, shell.exitstatus)
 
+    @unittest.skipUnless(testutil.TEST_RUN_JOBS, 'skipping tests that would run jobs')
+    def test_bundledDepends_name_with_whitespaces(self):
+        # upload a tar.gz file with spaces in its name
+        bundle_name = "test bundle with spaces.tar.gz"
```

Comment:
Since the content of this file isn't valid tar/gzip data (I presume), wouldn't dx-unpack fail even if it were fixed to handle filenames correctly?